docs/how-to/running-the-tests.md

```diff
@@ -0,0 +1,52 @@
+# How To - Running the tests
+
+## Introduction
+
+This guide shows you how to run both the unit and UI tests for an iOS app.
+
+## Where are the tests?
+
+The tests are located under the `Tests` and `UITests` folders respectively, found on the left hand side in the project directory in Xcode:
+
+<div style="width: 100%; max-width: 800px; margin-left: auto; margin-right: auto;">
+
+![Screenshot showing the project directory in Xcode](assets/running-the-tests/project-directory.png)
+
+</div>
+
+## How to run the tests
+
+To run all the tests together, click and press the play button in the top left of the toolbar and select `Test`. Xcode will automatically build and run all tests.
```

Comment:
Is "click and hold" clearer than "click and press"?

Reply:
That's what my brain was thinking, but not what I wrote down 🤦